src/utilities/helpers/focus-on-first-invalid-choice/index.ts

```diff
@@ -0,0 +1,14 @@
+import { type Ref } from 'vue';
+import UiMultipleChoices from '../../../components/organisms/UiMultipleChoices/UiMultipleChoices.vue';
+
+export default function focusOnInvalidChoice(invalidChoices: Ref<InstanceType<typeof UiMultipleChoices> | null>) {
+  const focusInvalidInput = async () => {
+    if (!invalidChoices.value) return;
+
+    const { focusInvalidChoice } = invalidChoices.value;
+
+    focusInvalidChoice();
+  };
+
+  return { focusInvalidInput };
+}
```

Comment:
I'm confused about this recurrence... 
So there's a function called `A`, which has inside a const variable named `B`, which is a function as well. In the body of `B` we call `A`... Aaah and we return `B` as the result of `A`. Is that correct?

Reply:
Well, why do we call A inside of B?